src/main/java/edu/harvard/iq/dataverse/harvest/client/HarvestingClient.java

```diff
@@ -214,6 +214,7 @@ public void setArchiveDescription(String archiveDescription) {
         this.archiveDescription = archiveDescription; 
     }
     
+    @Column(columnDefinition="TEXT")
     private String harvestingSet;
```

Comment:
Hmm, what was this before? 255 characters? Now it's unlimited?

Reply:
Yes. The accompanying flyway script adjusts the existing tables accordingly.

Reply:
Oh, another good call! That's something that came cherry-picked from my initial branch, and it is NOT needed in this branch. Will remove. 
(we want it to be unlimited, to be able to harvest from DataCite, where we'll be passing potentially very long lists of DOIs as "sets"; but it is not needed in this PR just yet).

Reply:
Removed in 8a361be. Thanks.